lntest/harness_miner.go

```diff
@@ -487,3 +495,89 @@ func (h *HarnessMiner) MineEmptyBlocks(num int) []*wire.MsgBlock {
 
 	return blocks
 }
+
+// SpawnTempMiner creates a temp miner and syncs it with the current miner.
+// Once miners are synced, the temp miner is disconnected from the original
+// miner and returned.
+func (h *HarnessMiner) SpawnTempMiner() *HarnessMiner {
+	require := require.New(h.T)
```

Comment:
What does this do exactly? From looking at the code it just creates a new instance of the `require` package itself. Was the intention to create a new instance of `testing.T` instead? Or why do we need to shadow the `require` package?

Reply:
yeah it's something chatGPT taught me😂, it just creates a new instance, so instead of calling `require(t, ...)` we can call `require(...)` without the `t` to shorten the call stack.

Reply:
Oh, I missed that part, didn't check the arguments. Okay, that's pretty useful I guess.